kodilla-spring/src/main/java/com/kodilla/spring/portfolio/Board.java

```diff
@@ -0,0 +1,18 @@
+package com.kodilla.spring.portfolio;
+
+import org.springframework.stereotype.Component;
+
+@Component
```

Comment:
To nie jest potrzebne - mamy konflikt. Beana definiujesz w klasie @Configuration